bundler/helpers/v2/run.rb

```diff
@@ -0,0 +1,30 @@
+require "bundler"
+require "json"
+
+$LOAD_PATH.unshift(File.expand_path("./lib", __dir__))
+$LOAD_PATH.unshift(File.expand_path("../v1/monkey_patches", __dir__))
```

Comment:
```suggestion
$LOAD_PATH.unshift(File.expand_path("../v2/monkey_patches", __dir__))
```

Reply:
oh wait this is intentional

Reply:
No worries, I flip flopped on that, I was nervous about hard forking monkey patches. It might make sense to go ahead and fork them but for now I just wanted to avoid breaking the loading.